android/app/src/main/java/com/reactnativenavigation/views/RnnToolBar.java

```diff
@@ -214,17 +214,16 @@ private void hideToolbar() {
     }
 
     public void setNavUpButton() {
-        setNavUpButton(null);
+        setNavUpButton(null, false);
     }
 
     @SuppressWarnings({"ConstantConditions"})
-    public void setNavUpButton(Screen screen) {
+    public void setNavUpButton(Screen screen, boolean isBack) {
```

Comment:
@jskuby Can we move the `setNavUpButton` call to `StyleHelper.updateStyles`? If so, seems like we determine if the back button needs to be displayed or not buy calling `getScreenStackSize() ` which is already public. Am I right?

In `RnnToolbar` we can grab a the `BaseReactActivity` instance by calling `getContext()`.
wdyt?